docs/integrations/sources/postgres/postgres-troubleshooting.md

```diff
@@ -146,14 +151,151 @@ The connector waits for the default initial wait time of 5 minutes (300 seconds)
 
 If you know there are database changes to be synced, but the connector cannot read those changes, the root cause may be insufficient waiting time. In that case, you can increase the waiting time (example: set to 600 seconds) to test if it is indeed the root cause. On the other hand, if you know there are no database changes, you can decrease the wait time to speed up the zero record syncs.
 
-### (Advanced) WAL disk consumption and heartbeat action query
+### (Advanced) Resolving sync failures due to WAL disk consumption
```

Comment:
This title change will cause a problem, because the Airbyte UI links directly to the text. We should replace the link with a persistent anchor link. This will require a change to the connector though, separately.